test/backup_transform.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python
```

Comment:
I guess there's no harm in splitting this out, but I'm surprised that it's necessary to work around test timeouts. We have tests that take various amounts of time. Is there really a hard-coded per-test-file timeout somewhere?

Reply:
When we run the python tests locally the timeout is set to 10 minutes. On Travis it is 8 minutes.

Reply:
Alright. I guess it's healthier overall to keep each file doing a reasonable amount of work.